### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
 ## Features
 
-Tide provides the following functionality:
-* Interactively view media such as high-resolution imagery and video
-* Receive content from remote sources such as laptops / desktops or
-  high-performance remote visualization machines using the
-  [Deflect library](https://github.com/BlueBrain/Deflect.git)
+Tide provides the following features:
+* Interactively view, present and collaborate on media such as high-resolution
+  imagery and movies.
```
Comment (rdumusc) [inline README.md]: imagery always sounded a bit strange to me, just images?

File: README.md [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
+* Receive and interact with content as pixel streams from remote sources such as
```
Comment (rdumusc) [inline README.md]: replace 'as pixel streams' with 'streamed'

File: doc/Documentation.md [hunk 3]
```diff
@@ -1,22 +1,38 @@
...
-  * Tide: The main application.
-  * LocalStreamer: Used by Tideto generate content from separate processes
-    (sandboxing).
-* tests: Unit tests
+  * core: The core library contains utilities, helpers and shared classes that
+          are used by the master and wall applications.
+  * master: The master library contains the control GUI and other
+            application-control logics.
+  * wall: The wall library contains the QML2 rendering engine for the supported
+          content types and synchronizations between other wall processes.
```
Comment (rdumusc) [inline doc/Documentation.md]: synchronizations -> their synchronization

File: tide/master/StateSerializationHelper.cpp [hunk 4]
```diff
@@ -169,8 +169,8 @@ void StateSerializationHelper::_validateContents( DisplayGroup& group ) const
                 contentWindow->getID().toString().toLocal8Bit().constData( ));
             continue;
         }
-        // PixelStreams are not supported yet, don't restore them.
-        // This feature will be implemented in DISCL-6
+
+        // PixelStreams are not supported, don't restore them.
```
Comment (rdumusc) [inline tide/master/StateSerializationHelper.cpp]: :-)

**Comment by rdumusc** [comment]
+1 - having a short "usage" section on the Readme would also be a nice addition, with an indication to refer to the UserGuide for more details (I'm not sure if people are aware that it exists).

**Comment by tribal-tec** [comment]
Updated

**Comment by rdumusc** [comment]
+1

**Comment by tribal-tec** [comment]
retest this please

**Pull request merged**